src/options/mod.rs

```diff
@@ -117,14 +120,17 @@ pub struct Options {
 
     /// The options to make up the styles of the UI and file names.
     pub theme: ThemeOptions,
+
+    /// Whether to read file names from stdin instead of the command-line
+    pub stdin: FilesInput,
 }
 
-impl Options {
+impl<'args> Options {
```

Comment:
Probably my fault but I believe this can be removed. 

I'll actually probably fix this (whole block I think) after work

Reply:
Ok I let you do that take your time